src/com/company/Board.java

```diff
@@ -0,0 +1,100 @@
+package com.company;
+
+import java.util.Arrays;
+
+public class Board implements IBoard {
```

Comment:
It was specifically mentioned to not use I prefix for interface. Make sure you are following the practices mentioned in the sessions to get your code reviewed. There is no point in mentioning the same things here that should have ideally been followed after the session. The code review is mostly for design and not these practices.

Reply:
my company follow that, I won't do this in next problem